cmd/rosa-support/create/subnets/cmd.go

```diff
@@ -37,36 +35,33 @@ func init() {
 		"region",
 		"",
 		"",
-		"Region of the security groups",
+		"Vpc region",
 	)
+	err := Cmd.MarkFlagRequired("region")
+	if err != nil {
+		logger.LogError(err.Error())
+		os.Exit(1)
+	}
 	flags.StringVarP(
-		&args.namePrefix,
-		"name-prefix",
+		&args.zones,
+		"zones",
```

Comment:
zones are specified as  a required field but not in the example above.
`  # Create a pair of subnets with prefix 'mysubnet-' in region 'us-east-2'
  rosa-support create subnets --name-prefix=mysubnet --region us-east-2 --vpc-id <vpc id>`,

can zones be called --availability-zones (following convention in ROSA)?